scripts/convert-to-safari-rules.ts

```diff
@@ -49,6 +49,26 @@ function isGreaterThan(thing: number, anotherThing: number, errormsg: string) {
   }
 }
 
+function isDomainValid(domain: string, starAllowed: boolean) {
+  function validate(d: string) {
+    if (!isValid(d)) {
+      throw new Error('not a valid domain = ' + domain);
+    }
+  }
+
+  const index = domain.indexOf('*');
+  const { isValid } = require('tldjs');
```

Comment:
Can you import this at the top of the file with other imports please?

Reply:
No!